pakyow-core/lib/pakyow/core/app.rb

```diff
@@ -198,8 +198,13 @@ class App
           secret: config.session.secret
         }
 
+        # set expiry if set
+        if expiry = config.session.expiry
+          opts[:expire_after] = config.session.expiry
```

Comment:
I think you might have meant to use the expiry variable you created in the line above.